webpack.config.js

```diff
@@ -60,14 +72,5 @@ module.exports = {
       chunks: ["popup"],
       inject: "body",
     }),
-    function () {
```

Comment:
Manifest is still adding to dist.

That function was used to change file path from dist/something to /something in copied manifest.json file in dist folder, now the main manifest.json use /something rather than dist/something. 

```js

//...
// From
"default_popup": "dist/popup.html"
// TO
"default_popup": "/popup.html"
//...

```


I don't find any use case to use dist/something instead of  /something in manifest.json that is sitting in our project's root folder.

so, we did this